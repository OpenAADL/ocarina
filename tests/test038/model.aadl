package Deployment
public
  data D
  end D;

private
  data E
  end E;
end Deployment;
