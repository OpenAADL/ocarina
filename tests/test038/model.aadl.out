model.aadl:1:09: "Deployment" conflicts with declaration in deployment.aadl:1:01
Cannot analyze AADL specifications
